src/app.module.ts

```diff
@@ -1,17 +1,30 @@
 import { getNodeEnv } from '@fullstacksjs/toolbox';
 import { Module } from '@nestjs/common';
 import { ConfigModule } from '@nestjs/config';
+import Joi from 'joi';
 import path from 'path';
 
-import configuration from './config/configuration';
+import { PrismaModule } from './prisma/prisma.module';
 
 @Module({
   imports: [
     ConfigModule.forRoot({
       cache: false,
+      isGlobal: true,
       envFilePath: [path.join(process.cwd(), `.${getNodeEnv()}.env`)],
-      load: [configuration],
+      validationSchema: Joi.object({
+        NODE_ENV: Joi.string()
+          .valid('production')
+          .valid('development')
+          .valid('test')
+          .required(),
+        PORT: Joi.number().required(),
+        DATABASE_URL: Joi.string()
```

Comment:
We should disable the linter warnings in exceptional cases.

Reply:
for each line or for the entire file?